sklearn/multioutput.py

```diff
@@ -1166,6 +1227,12 @@ class RegressorChain(MetaEstimatorMixin, RegressorMixin, _BaseChain):
 
         .. versionadded:: 1.2
 
+    base_estimator : estimator, default=None
+        Use `estimator` instead.
+        .. deprecated:: 1.6
+            `base_estimator` is deprecated and will be removed in 1.8.
+            Use `estimator` instead.
+
```

Comment:
```suggestion
    base_estimator : estimator, default="deprecated"
        Use `estimator` instead.

        .. deprecated:: 1.6
           `base_estimator` is deprecated and will be removed in 1.8.
            Use `estimator` instead.

```

Reply:
The comment above about `default="deprecated"` also applies here.

Reply:
`None` would mean that we would have an option internally at `RegressorChain` and `ClassifierChain` to select a default estimator. However, this is not the case. This is different from `BaggingClassifier` where `None` meant a `DecisionTreeClassifier`.

So the deprecation here is simpler.